src/info/author.rs

```diff
@@ -221,4 +234,39 @@ mod test {
             .value()
             .contains(&"80% Roberto Berto 240".to_string()));
     }
+    #[test]
+    fn test_author_info_value_alignment_with_three_authors() {
```

Comment:
I think that this test is a good candidate for a snapshot test, so it's easier for contributors to visualize the tested behavior.

Reply:
I'm afraid I'm not familiar with the snapshot tests, or what this means. Could you give me some pointers?

Reply:
Sure! You'll want to look into `insta` and `cargo-insta`. We have a few snapshot tests in our codebase using macros from `insta` that you can use as an example.

The basic idea of a snapshot test is that you give it only the actual value, and the snapshot test assertion will generate that value in a file for you to review and commit. I typically find these helpful for complex or styled outputs. So, for a snapshot test here, I'd recommend asserting the entire author info stat converted to a string. The snapshot that gets generated will act as a preview of how it will look in the CLI.

Reply:
I'm afraid I still do not understand this. Can you give me some pointers to these existing snapshot tests, and how I'm supposed to run them?

Reply:
OK, did a quick search of our codebase and it turns out that snapshot tests aren't as common in this project as I thought. There's a JSON snapshot test [here](https://github.com/o2sh/onefetch/blob/5b1ef678c2b3f86e90a848cf20fd236d230b3110/tests/repo.rs#L52-L58), which automatically generated [this snapshot](https://github.com/o2sh/onefetch/blob/5b1ef678c2b3f86e90a848cf20fd236d230b3110/tests/snapshots/repo__repo.snap). I might try to add the snapshot test myself later, but if you want to take one more look I'd recommend starting with [insta's docs](https://insta.rs/docs/quickstart/). You'd probably want either `assert_snapshot` or `assert_display_snapshot`. You can run them like any other test with `cargo test`, but you might want to install `cargo-insta` so that you can use the `cargo insta test` and `cargo insta review` subcommands.

Reply:
My preference would to not block this PR on this new test type.

Reply:
OK, sorry, I didn't realize that you had basically copied similar tests. You're right that this PR shouldn't be blocked, and all of those tests should be updated in a separate PR instead.